packages/angular/README.md

```diff
@@ -1,7 +1,21 @@
-# Carbon Charts Angular
-
-Carbon Charts Angular is a thin Angular wrapper around the vanilla JavaScript `@carbon/charts`
-component library. This prerelease is for Angular 18 and higher.
+<p align="center">
+	<a href="https://charts.carbondesignsystem.com/">
+		<img src="../../assets/dashboard.png" alt="Carbon Charts" />
+	</a>
+	<h3 align="center">Carbon Charts Angular</h3>
+	<p align="center">
+		A component library of 26 charts for Angular 18.
+		<br /><br />
+		<a href="https://www.npmjs.com/package/@carbon/charts">
+			<img src="https://img.shields.io/npm/v/@carbon/charts.svg" />
+		</a>
+		<img alt="semantic-versioning" src="https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--ver-e10079.svg" />
+		<img alt="semantic-versioning" src="https://img.shields.io/badge/downloads-+60k%2Fweek-green" />
+		<a href="https://discord.gg/J7JEUEkTRX">
+	    		<img src="https://img.shields.io/discord/689212587170201628?color=5865F2" alt="Chat with us on Discord">
+	  	</a>
+	</p>
+</p>
```

Comment:
Makes the read me more visually interesting over what we had especially important for npmjs page.